cmd/global-heal.go

```diff
@@ -72,11 +72,11 @@ func getLocalBackgroundHealStatus() madmin.BgHealState {
 	if !ok {
 		return madmin.BgHealState{}
 	}
-
+	// FIXME:
 	return madmin.BgHealState{
 		ScannedItemsCount: bgSeq.getScannedItemsCount(),
 		LastHealActivity:  bgSeq.lastHealActivity,
-		NextHealRound:     UTCNow().Add(durationToNextHealRound(bgSeq.lastHealActivity)),
+		NextHealRound:     UTCNow(),
```

Comment:
Perhaps this should be UTCNow() + crawlCycle? @klauspost

Reply:
Well, then it would always be 5 minutes in the future. Not sure I like that more. The field doesn't make much sense any more.

Reply:
yeah that would be more accurate than always showing current time